ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
I don't think we'll agree on this- we should probably try to get a second opinion: @lbustelo , @ellisonbg , or @jasongrout  , what are your opinions on adding `border` to the mutable `Layout` object?